semantics/language/execution/expr/assignment.k

```diff
@@ -56,7 +56,7 @@ module C-EXPR-ASSIGNMENT
           [structural]
      rule (.K => UNDEF("EEA1",
                "assignment from non-exactly-overlapping overlapping read.",
-               "6.5.16.2"))
+               "6.5.16.1:3, J.2:1 item 54"))
```

Comment:
@dwightguth , please carefully check these ones where @msaxena2 changed the section/paragraph.